site/docs/reference/Connectors/capture-connectors/notion.md

```diff
@@ -69,14 +70,14 @@ so many of these properties aren't required.
 | **`/credentials`** | Authenticate using | Pick an authentication method. | object | Required |
 | **`/credentials/auth_type`** | Authentication type | Set to `token` for manual authentication | string | Required |
 | `/credentials/token` | Access Token | Notion API access token | string | |
-| **`/start_date`** | Start Date | UTC date and time in the format 2017-01-25T00:00:00.000Z. Any data before this date will not be replicated. | string | Required |
+| `/start_date` | Start Date | UTC date and time in the format YYYY-MM-DDTHH:MM:SS.000Z. During incremental sync, any data generated before this date will not be replicated. If left blank, the start date will be set to 2 years before the present date. | string |  |
```

Comment:
I looked over the code, and it was not obvious to me that in full refresh mode we do not respect this `start_date`. To me it seems that `start_date` applies to bindings even if they have `sync_mode: full_refresh` 🤔